docs/learn/avalanche/avalanche-consensus.md

```diff
@@ -1,45 +1,81 @@
+the shared ledger. This agreement is reached through a specific process, a consensus protocol,
+that ensures that everyone sees the same information and that the information is accurate and trustworthy.
+
+**Avalanche Consensus** is a new consensus protocol that is scalable, robust, and decentralized. It
+combines features of both classical and Nakamoto consensus mechanisms to achieve high throughput, 
+fast finality, and
+energy efficiency. For the whitepaper, see [here](https://www.avalabs.org/whitepapers). 
+
+Key Features Include: 
+
+- Speed: Avalanche consensus provides sub-second, immutable finality, ensuring that transactions are
+quickly confirmed and irreversible.
+- Scalability: Capable of a peak throughput of up to 20,000 transactions per second with a latency
+of less than half a second.
+- Energy Efficiency: Unlike other popular consensus protocols, participation in Avalanche consensus
+is not computationally intensive nor expensive. 
+- Adaptive Security: Avalanche consensus is designed to resist various attacks, including Sybil
+attacks, distributed denial-of-service (DDoS) attacks, and collusion attacks. Its probabilistic
+nature ensures that the consensus outcome converges to the desired state, even when the network
+is under attack. 
 
 ![Consensus Comparison](/img/Consensus-protocol-comparison.png)
 
-Protocols in the Avalanche family operate through repeated sub-sampled voting. When a
+## Conceptual Overview
+
+Consensus protocols in the Avalanche family operate through repeated sub-sampled voting. When a
 [validator](http://support.avalabs.org/en/articles/4064704-what-is-a-blockchain-validator) is
 determining whether a
 [transaction](http://support.avalabs.org/en/articles/4587384-what-is-a-transaction) should be
-accepted, it asks a small, random subset of validators on their preferences. Each queried validator
-replies with the transaction that it prefers, or thinks should be accepted. The transaction that is
-received from the reply might be the same transaction indicating agreement, or it might be another
-conflicting transaction.
+accepted, it asks a small, random subset of validators for their preference. Each queried validator
+replies with the transaction that it prefers, or thinks should be accepted.
+
+:::note
+
+Consensus will never include a transaction that is determined to be **invalid**. For example, if you
+were to submit a transaction to send 100 AVAX to a friend, but your wallet only has 2 AVAX, this transaction
+is considered **invalid** and will not participate in consensus. 
+
+:::
+
+If a majority of the validators sampled reply with the same preferred
+transaction, this becomes the preferred choice of the group sampled, including the validator that asked.
+In the future, this validator and all those who have participated in at least one round of sampling
+will reply with the transaction preferred by the majority.
```

Comment:
@aaronbuchwald : 
This is not quite accurate. Does this need to be changed in this PR? The previous version was not quite correct either.

Votes are performed through vertices/blocks as opposed to transactions.

Additionally, validators maintain a confidence counter and do not immediately change their preference after a single poll in favor of a different container (transaction in the current text).

Reply:
@martineckardt do you have any materials written for the academy that could help me rephrase this section?

Reply:
This doesn't need to be changed in this PR, this is a great first step

Reply:
@aaronbuchwald meaning this comment doesn't need to be addressed and we can include the current change in this PR, or I should remove these changes and push the rest?

Reply:
Personally, I'd just omit this line entirely